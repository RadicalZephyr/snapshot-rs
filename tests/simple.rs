#![feature(proc_macro)]

extern crate snapshot;

mod test {
    use snapshot::snapshot;

    #[snapshot]
    fn simple_snapshot() -> i32 {
        let x = 1;
        x
    }
}
