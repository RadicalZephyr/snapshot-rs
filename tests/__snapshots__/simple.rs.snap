{
  "simple::test::simple_snapshot": {
    "file": "tests/simple.rs",
    "module_path": "simple::test",
    "test_function": "simple_snapshot",
    "recorded_value": 1
  }
}